backend/src/validation.rs

```diff
@@ -47,6 +51,10 @@ impl fmt::Display for ValidationResultCode {
             ValidationResultCode::AboveThreshold => write!(f, "Above threshold"),
             ValidationResultCode::EqualInput => write!(f, "Equal input"),
             ValidationResultCode::IncorrectCandidatesList => write!(f, "Incorrect candidates list"),
+            ValidationResultCode::MissingRecounts => write!(f, "Missing recounts"),
+            ValidationResultCode::IncorrectDifference => write!(f, "Incorrect difference"),
+            ValidationResultCode::WrongDifferences => write!(f, "Wrong differences"),
```

Comment:
The difference between the code `IncorrectDifference` and `WrongDifferences` was not immediately obvious to me.

Suggestion: change `WrongDifferences` to `ConflictingDifferences`.

Reply:
Done in ed1c100539a85fb761d3dd72d650e5481f6854ea! Was struggling to find an understandable code for this